src/Controller/UserController.php

```diff
@@ -17,10 +17,8 @@ public function __construct(ManagerRegistry $doctrine)
     {
         $this->doctrine = $doctrine;
     }
-
-    /**
-     * @Route("/api/users/{user}", name="user_put",  methods={"PUT"})
-     */
+  
```

Comment:
There's extra spaces here; the newline should not have any space characters in it